### PR TITLE
Experiment tab: disable and clear controls when disconnected

### PR DESCRIPTION
**Pull request by mattclarke**
Changes to the setup_exp:
 * disable controls when view-only and disconnected
 * clear the text contents when disconnected
 * Removed send data textfield as that is not needed

### CODE REVIEW

File: nicos_ess/loki/gui/setup_exp.py [hunk 1]
```diff
@@ -151,10 +161,8 @@ def _getProposalInput(self):
         emails = emails.split('\n') if emails else []
         if local and local not in emails:
             emails.append(local)
-        dataEmails = self.dataEmails.toPlainText().strip()
-        dataEmails = dataEmails.split('\n') if dataEmails else []
         errorbehavior = 'abort' if self.errorAbortBox.isChecked() else 'report'
-        return prop, title, users, local, emails, dataEmails, errorbehavior
+        return prop, title, users, local, emails, [], errorbehavior
```
Comment (umithardal) [inline nicos_ess/loki/gui/setup_exp.py:166]: Why do we need to return an empty list?

File: nicos_ess/loki/gui/setup_exp.py [hunk 2]
```diff
@@ -122,12 +124,20 @@ def on_client_connected(self):
...
     def setViewOnly(self, value):
         for button in self.buttonBox.buttons():
             button.setEnabled(not value)
-        self.queryDBButton.setEnabled(not value)
+
+        for control in self._text_controls:
+            control.setEnabled(not value)
+
+        self.notifEmails.setEnabled(not value)
+        self.errorAbortBox.setEnabled(not value)
```
Comment (umithardal) [inline nicos_ess/loki/gui/setup_exp.py:140]: So the value is a boolean?

File: nicos_ess/loki/gui/setup_exp.py (hunk 1 above)
Comment (mattclarke) [reply nicos_ess/loki/gui/setup_exp.py:166]: Good point - thanks 👍

File: nicos_ess/loki/gui/setup_exp.py (hunk 2 above)
Comment (mattclarke) [reply nicos_ess/loki/gui/setup_exp.py:140]: Yeah

Comment (umithardal) [reply nicos_ess/loki/gui/setup_exp.py]: Ok last commit solved my problem for understanding the context. Still negating the boolean in the function call a bit interesting, but quite creative. Thanks.

**Review by umithardal — APPROVED** [review]

**Pull request merged**